content/en/docs/collector/internal-telemetry.md

```diff
@@ -136,7 +136,7 @@ journalctl | grep otelcol | grep Error
 
 ## Types of internal telemetry
 
-The OpenTelemetry Collector aims to be a model of observable service by clearly
```

Comment:
This one should be undone. The use of `clearly` here is to explain how the collector isn't merely emitting telemetry, but it is doing so in a way believed to be ideal so that it can be used as a model for other systems.

Reply:
Thanks for the feedback, I will make the update.